media/css/firefox/firstrun/firstrun-quantum.scss

```diff
@@ -64,26 +63,24 @@ body {
     z-index: 10;
     margin: auto;
 
-    @supports (display: grid) {
-        @media #{$mq-medium} {
-            display: grid;
-            grid-column-gap: 20px;
-            grid-template-columns: 390px 1fr;
-            margin: 0 auto;
-            max-width: (390px * 2) + 20px;
-        }
+    @media #{$mq-medium} {
+        display: grid;
+        grid-column-gap: 20px;
+        grid-template-columns: 390px 1fr;
+        margin: 0 auto;
+        max-width: (390px * 2) + 20px;
+    }
 
-        // a bit more space between once the screen is wide enough
-        @media screen and (min-width: 800px) {
-            max-width: (390px * 2) + 60px;
-            grid-column-gap: 80px;
-            margin-top: calc((100vh - 540px) / 2);
-        }
+    // a bit more space between once the screen is wide enough
+    @media screen and (min-width: 800px) {
+        max-width: (390px * 2) + 60px;
```

Comment:
Didn't catch this the first time around, but this `max-width` exceeds the media query by a bit, causing a portion of the iframe to bleed off the edge (between 800px and ~828px wide).